src/main/java/com/rultor/agents/github/qtn/QnIfPull.java

```diff
@@ -77,16 +77,43 @@ public Req understand(final Comment.Smart comment,
         final URI home) throws IOException {
         final Issue.Smart issue = new Issue.Smart(comment.issue());
         final Req req;
-        if (issue.isPull()) {
-            req = this.origin.understand(comment, home);
-        } else {
+        if (QnIfPull.isNotPull(issue)) {
             new Answer(comment).post(
                 false,
                 QnIfPull.PHRASES.getString("QnIfPull.not-pull-request")
             );
-            req = Req.EMPTY;
+            req = Req.DONE;
```

Comment:
@original-brownbear 
I don't know why QnIfPull returned EMPTY Req at the beginning, because according to how it used QnIfPull should return Req.DONE in case of some incorrectness with issue (like for QnByArchitect). Why we have to proceed
 flow forward if command not possible (see QnFirstOf )